src/backend/vulkanll/src/lib.rs

```diff
@@ -45,14 +66,117 @@ pub struct SwapChain {
 
 impl core::SwapChain for SwapChain {
     type B = Backend;
+
+    fn present(&mut self) {
+        unimplemented!()
+    }
 }
 
-pub struct Instance {
+struct InstanceInner(pub ash::Instance<V1_0>);
+impl Drop for InstanceInner {
+    fn drop(&mut self) {
+        unsafe { self.0.destroy_instance(None); }
+    }
+}
+
+const SURFACE_EXTENSIONS: &'static [&'static str] = &[
```

Comment:
You can also use Ash's extensions names but it is not necessary. `Surface::name().as_ptr()` would give you a const* i8.